pyOCD/tools/gdb_server.py

```diff
@@ -44,6 +44,10 @@ def __init__(self):
         self.args = None
         self.gdb_server_settings = None
         self.echo_msg = None
+        self.input_args = None
```

Comment:
The variable self.args is for passing arguments to the GDBServer class, while input_args are used to override the args passed on the command line.  I'll modify how this is handled to make it more clear.